notebook/mlops.ipynb

```diff
@@ -4,28 +4,57 @@
    "cell_type": "markdown",
```

Comment:
<p>I'm not sure this is correct - this code will complete once the API output resource is available.  If there is a canary deployment - then the next section that waits for the stack to complete will take between 5-10 mins.  The 5 minutes is the time for the CodeDeploy to complete, and then its a few extra minutes to delete the old endpoints and cleanup.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws-samples/amazon-sagemaker-safe-deployment-pipeline/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>